inst/rmarkdown/templates/springer/skeleton/skeleton.Rmd

```diff
@@ -1,63 +1,380 @@
-- key
-- dictionary
-- word
+  - key
+  - dictionary
+  - word
 
-#PACS: 
-#- PAC1
-#- superPAC
+PACS:
+  - D8
+  - H51
     
-MSC:
-- MSC code 1
-- MSC code 2
+# MSC:
+#   - 35A01
+#   - 65L10
 
+# sample for unstructured abstract 
+# abstract: |
+#   The abstract serves both as a general introduction to the topic and as a brief, non-technical summary of the main results and their implications. Authors are advised to check the author instructions for the journal they are submitting to for word limits and if structural elements like subheadings, citations, or equations are permitted.
+
+# Sample for structured abstract
 abstract: |
-  The text of your abstract.  150 -- 250 words.
+  **Purpose**: The abstract serves both as a general introduction to the topic and as a brief, non-technical summary of the main results and their implications. The abstract must not include subheadings (unless expressly permitted in the journal's Instructions to Authors), equations or citations. As a guide the abstract should not exceed 200 words. Most journals do not set a hard limit however authors are advised to check the author instructions for the journal they are submitting to.
+
+  **Methods:** The abstract serves both as a general introduction to the topic and as a brief, non-technical summary of the main results and their implications. The abstract must not include subheadings (unless expressly permitted in the journal's Instructions to Authors), equations or citations. As a guide the abstract should not exceed 200 words. Most journals do not set a hard limit however authors are advised to check the author instructions for the journal they are submitting to.
+
+  **Results:** The abstract serves both as a general introduction to the topic and as a brief, non-technical summary of the main results and their implications. The abstract must not include subheadings (unless expressly permitted in the journal's Instructions to Authors), equations or citations. As a guide the abstract should not exceed 200 words. Most journals do not set a hard limit however authors are advised to check the author instructions for the journal they are submitting to.
+
+  **Conclusion:** The abstract serves both as a general introduction to the topic and as a brief, non-technical summary of the main results and their implications. The abstract must not include subheadings (unless expressly permitted in the journal's Instructions to Authors), equations or citations. As a guide the abstract should not exceed 200 words. Most journals do not set a hard limit however authors are advised to check the author instructions for the journal they are submitting to.}
 
 bibliography: bibliography.bib
-biblio-style: spphys
-# bibstyle options spbasic(default), spphys, spmpsci
+
 output: rticles::springer_article
 ---
 
-# Introduction {#intro}
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE,
+                      out.extra = "")  # This forces knitr to label all figures.
```

Comment:
It's kind of a workaround. Knitr will not label figures that have no caption; adding `out.extra` to be anything forces knitr to add labels. It's not required but it can be confusing if its not there.  It's on the AGU template too.

Reply:
Oh interesting. This is indeed a required workaround to get **knitr** to produce LaTeX code directly and not let pandoc handles image. 

Probably something we can improve in **knitr** part of https://github.com/yihui/knitr/issues/1991

Let's leave it then. Thanks for the explanation